lib/contentful-management.ts

```diff
@@ -61,13 +61,22 @@ function createClient(
     defaults?: DefaultParams
   }
 ): PlainClientAPI
+function createClient(
+  params: ClientOptions,
+  opts: {
+    type: 'plain'
+    alphaFeature: string[]
```

Comment:
I saw in some places we renamed to `alphaFeatures` (plural) but here is still singular, is it intentional?